gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskRepository.java

```diff
@@ -23,13 +23,13 @@
 @Repository
 public interface TaskRepository extends JpaRepository<Task, UUID> {
 
-    @Query("SELECT task FROM Task task LEFT JOIN FETCH task.processFiles WHERE task.id = :id")
+    @Query("SELECT task FROM Task task LEFT JOIN FETCH task.processEvents LEFT JOIN FETCH task.processFiles WHERE task.id = :id")
     Optional<Task> findByIdWithProcessFiles(@Param("id") UUID id);
 
-    @Query("SELECT task FROM Task task LEFT JOIN FETCH task.processFiles WHERE task.timestamp = :timestamp")
+    @Query("SELECT task FROM Task task LEFT JOIN FETCH task.processEvents LEFT JOIN FETCH task.processFiles WHERE task.timestamp = :timestamp")
     Optional<Task> findByTimestamp(@Param("timestamp") OffsetDateTime timestamp);
 
-    @Query("SELECT task FROM Task task JOIN FETCH task.processFiles " +
+    @Query("SELECT task FROM Task task LEFT JOIN FETCH task.processEvents JOIN FETCH task.processFiles " +
```

Comment:
You can use java text blocks now

like this

![Capture d’écran de 2023-10-11 15-52-39](https://github.com/farao-community/gridcapa-task-manager/assets/116628858/3fb4e2d9-1c65-4ff8-8da8-bea1b31de52b)

or like this

![Capture d’écran de 2023-10-11 15-57-00](https://github.com/farao-community/gridcapa-task-manager/assets/116628858/b88e695a-7c0c-4b7f-9bc9-8dea1fd16cc6)

Reply:
doesn't pass check style